app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -1405,7 +1405,10 @@ var RepositoryDatatable = (function(global) {
   }
 
   function destroy() {
-    TABLE = {};
+    if (TABLE !== null) {
```

Comment:
and here you can check if(!_.isUndefined(TABLE))

Reply:
isn't comparison to null more clear? Variable should be always defined here I assume